projects/angular/src/forms/combobox/options.ts

```diff
@@ -44,7 +44,11 @@ let nbOptionsComponents = 0;
 
     <!-- Rendered if data set is empty -->
     <div *ngIf="emptyOptions">
-      <span class="clr-combobox-options-empty-text">
+      <span
+        class="clr-combobox-options-empty-text"
+        id="{{ optionSelectionService.noResultsElementId }}"
```

Comment:
```suggestion
        [id]="optionSelectionService.noResultsElementId"
```

This can be simplified.